vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -202,7 +202,7 @@
 
       <ComplexType Name="RecursiveHierarchyType">
         <Property Name="NodeProperty" Type="Edm.PropertyPath" Nullable="false">
-          <Annotation Term="Core.Description" String="Property holding the hierarchy node value" />
+          <Annotation Term="Core.Description" String="Property holding the hierarchy node value. It MUST be of type `Edm.String`." />

```

Comment:
So can we change the type of parameters `Node`, `Ancestor`, `Descendant`, and `Sibling` also to `Edm.String`?